example/regions/index.html

```diff
@@ -29,6 +29,13 @@
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/styles/default.min.css">
         <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/highlight.min.js"></script>
         <script>hljs.initHighlightingOnLoad();</script>
+
+        <!-- Fix strange default css that makes handles hard to grab when the region is small -->
+        <style>
+            .wavesurfer-region > handle {
+                width: 2px !important;
```

Comment:
if this is happening in the example, perhaps it's something worth putting in the library itself?  probably as a `minWidth` property in the handle css...

Reply:
Changed default to 2px in the plugin. I'm not sure why it was 1% as default. Maybe I'm missing something. If you think it would be better, we can keep it as it was but add minWidth: 1px so the handle cant disappear